spec/Vehicle/Vehicle.vspec

```diff
@@ -55,6 +55,7 @@ VehicleIdentification.VIN:
   datatype: string
   type: attribute
   description: 17-character Vehicle Identification Number (VIN) as defined by ISO 3779.
+  pattern: ^([0-9A-HJ-NPR-Z]{3})([0-9A-HJ-NPR-Z]{6})([0-9A-HJ-NPR-Z]{4}[0-9]{4})$
```

Comment:
I like the idea. Lets discuss the general idea of supporting/allowing a `pattern`keyword at the VSS meeting today.

If we agree that it is a good idea for cases like this then we need to look into details:
* When should it be allowed, I assume only for `string` and `string[]` signals and struct properties.
* Which syntax is supported - I think it would be good if we could specify something specific like that the pattern should comply with [POSIX Basic Regular Expressions ](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap09.html#tag_09_03). At least for the standard catalog there is a value in selecting a standard/subset which is widely supported in different libraries so that it easily can be verified in VISS (Go), Kuksa (Rust/Python) and vss-tools (Python) 

With those questions answered it would be easy to document this. One could also when discussing this think about #771. For VIN it is unlikely that we will have more than one signal with VIN-syntax, but for other use cases one could think of that we want to reuse the pattern, and if we introduce/refactor format-handling then one could in addition consider allowing pattern also for formats, for instance to give  regexp that complies with `iso8601` timestamps.

Reply:
Thank you Erik.
I will add recommended validations in the related VSS-Tools PR, where @sschleemilch also advised to provide similar validation of the field in the VSS model.